### PR TITLE
Backend: working streaming implementation, very basic streaming (auto restarts)

### PR DESCRIPTION
**Pull request by map34**
## Overview

working streaming implementation, very basic streaming (auto restarts)

### Notes

* Note that for some reason OpenCV recognizes a small chunk of the video. Need to figure it out exactly why.

* All of the videos are stored as raw bytes in Kafka. OpenCV can easily serialize + deserialize the components easily.

* We need to somehow divide the requests into multiple parts (like how youtube does it). Not just streaming at once. We should also implement expiration to avoid hogging the browser.

## Testing Instructions
* Please rebuild your images
* Send the video to the producer 
-> `docker-compose -f infra/docker_dev/docker-compose.yml run oceanhub_server bash -c "source activate TEST && python backend/server/manage.py shell"`
-> test_producer_video()
* Go to http://localhost:5000/videos/ to see it subscribing to the producer (in this case, the /videos/  endpoint acts as a consumer)

### CODE REVIEW

File: backend/server/oceanhub/scripts/kafka_example.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python
```
Comment (lsetiawan) [inline backend/server/oceanhub/scripts/kafka_example.py:1]: This python code is not actually doing anything, correct?

**Review by lsetiawan — APPROVED** [review]
I got streaming frames! This is so cool! :smile: You are brilliant @map34

File: backend/server/oceanhub/scripts/kafka_example.py (hunk 1 above)
Comment (map34) [reply backend/server/oceanhub/scripts/kafka_example.py:1]: You are correct :)

**Comment by lsetiawan** [comment]
I say that this should be merged! :smile:

**Pull request merged**